### PR TITLE
a first version of the biography

### PR DESCRIPTION
**Pull request by CaptainSifff**

### CODE REVIEW

File: summarised_competencies.md [hunk 1]
```diff
@@ -779,46 +779,13 @@ a direction determined by new incentives created by scientific journals and libr
...
-Frank Löffler rewrote numerous parts to be actually legible, and helped with preparation for the final steps of a de-RSE position paper.
-Michele Martone wrote the first draft of the environmental sustainability section.
-Moritz Schwarzmeier drafted the categorisation of the specialisations.
-Heidi Seibold contributed the idea and started everything.
-Jan Philipp Thiele drafted initial parts of the technical pillar of the RSE competencies, and represented the project on numerous discussions.
-Harald von Waldow contributed to initial drafts of the Masters program and contributed his knowledge to the explainability of computer simulations.
-Samantha Wittke contributed the parts on CodeRefinery and how to reach out to new RSEs.
-Florian Goth has the pleasure of being grateful to all collaborators in this project for contributing their time and knowledge into this project!
+# teachingRSE biography
+The teachingRSE project has been working on the associated biger publication in FIXME for the past two years.
```
Comment (mhagdorn) [inline summarised_competencies.md:783]:
```suggestion
The teachingRSE project has been working on the associated bigger publication in FIXME for the past two years.
```
fix typo

File: summarised_competencies.md [hunk 2]
```diff
@@ -779,46 +779,13 @@ a direction determined by new incentives created by scientific journals and libr
...
+But how did it all start?
+There was Heidi Seibold who came up with the original idea for the deRSE23 workshop in Paderborn.
```
Comment (mhagdorn) [inline summarised_competencies.md:785]: a little less conversational maybe
```suggestion
The project was initiated by Heidi Seibold who came up with the original idea for the deRSE23 workshop in Paderborn.
```

File: summarised_competencies.md [hunk 3]
```diff
@@ -779,46 +779,13 @@ a direction determined by new incentives created by scientific journals and libr
...
+Heidi was joined by Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele, and Samantha Wittke to organise the initial deRSE23 workshop.
+Over the course of further workshops like the un-deRSE23 workshop in Jena, and the deRSE24 in Würzburg,
+the content of the paper was further refined and we are grateful to all particpants!
+Also the group gained more recognition, and the teachingRSE project is now an official working group
```
Comment (mhagdorn) [inline summarised_competencies.md:789]:
```suggestion
The group gained more recognition, and the teachingRSE project is now an official working group 
```

File: summarised_competencies.md [hunk 4]
```diff
@@ -779,46 +779,13 @@ a direction determined by new incentives created by scientific journals and libr
...
-Florian Goth has the pleasure of being grateful to all collaborators in this project for contributing their time and knowledge into this project!
+# teachingRSE biography
+The teachingRSE project has been working on the associated biger publication in FIXME for the past two years.
+But how did it all start?
+There was Heidi Seibold who came up with the original idea for the deRSE23 workshop in Paderborn.
+Heidi was joined by Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele, and Samantha Wittke to organise the initial deRSE23 workshop.
+Over the course of further workshops like the un-deRSE23 workshop in Jena, and the deRSE24 in Würzburg,
+the content of the paper was further refined and we are grateful to all particpants!
+Also the group gained more recognition, and the teachingRSE project is now an official working group 
+in de-RSE and in the special interest group on research software of the Gesellschaft für Informatik.
```
Comment (mhagdorn) [inline summarised_competencies.md:790]:
```suggestion
of the de-RSE society and the special interest group on research software of the Gesellschaft für Informatik.
```

**Review by mhagdorn — CHANGES_REQUESTED** [review]
some minor bits and bobs

File: summarised_competencies.md [hunk 5]
```diff
@@ -779,46 +779,13 @@ a direction determined by new incentives created by scientific journals and libr
...
-Jan Philipp Thiele drafted initial parts of the technical pillar of the RSE competencies, and represented the project on numerous discussions.
-Harald von Waldow contributed to initial drafts of the Masters program and contributed his knowledge to the explainability of computer simulations.
-Samantha Wittke contributed the parts on CodeRefinery and how to reach out to new RSEs.
-Florian Goth has the pleasure of being grateful to all collaborators in this project for contributing their time and knowledge into this project!
+# teachingRSE biography
+The teachingRSE project has been working on the associated biiger publication in FIXME for the past two years.
+But how did it all start?
+There was Heidi Seibold who came up with the original idea for the deRSE23 workshop in Paderborn.
+Heidi was joined by Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele, and Samantha Wittke to organise the initial deRSE23 workshop.
+Over the course of further workshops like the un-deRSE23 workshop in Jena, and the deRSE24 in Würzburg,
```
[the remark this answers is not in the document]
Comment (mhagdorn) [reply summarised_competencies.md:787]: What about adding

In addition to the workshops the ideas were further developed  during weekly meetings led by Florian Goth.

[the remark this answers is not in the document]
Comment (CaptainSifff) [reply summarised_competencies.md]: added text trying to resolve both.

**Pull request merged**